storage/pebble/config.go

```diff
@@ -46,19 +45,15 @@ func DefaultPebbleOptions(logger zerolog.Logger, cache *pebble.Cache, comparer *
 		l.FilterPolicy = bloom.FilterPolicy(MinLookupKeyLen)
 		l.FilterType = pebble.TableFilter
 
-		if i > 0 {
-			// L0 starts at 2MiB, each level is 2x the previous.
-			l.TargetFileSize = opts.Levels[i-1].TargetFileSize * 2
-		}
-		l.EnsureDefaults()
+		l.EnsureL0Defaults()
 	}
 
 	// TODO(rbtz): benchmark with and without bloom filters on L6
 	// opts.Levels[6].FilterPolicy = nil
 
 	// Splitting sstables during flush allows increased compaction flexibility and concurrency when those
 	// tables are compacted to lower levels.
-	opts.FlushSplitBytes = opts.Levels[0].TargetFileSize
+	opts.FlushSplitBytes = opts.TargetFileSizes[0]
```

Comment:
moved to main opts struct
https://github.com/cockroachdb/pebble/pull/4929/files#diff-4eec45b6a922746411c0f8aea183e30f77af1b0a5bea1688eaf343a5f9c23703R863